docs/source/models.rst

```diff
@@ -628,22 +605,17 @@ accepts the following data formats as input:
                                        location=location,
```

Comment:
@shivp950 Can we update the paragraph/description at the beginning of this docs section (`Deploy a python function model on Microsoft Azure ML`) to indicate that the `mlflow.azureml` module provides tools for building deployment images (deprecated) as well as creating new AKS / ACI deployments? This paragraph seems out-of-date and should be updated to reflect the new API.

cc @akshaya-a

Reply:
makes sense, will do